Doc/library/stdtypes.rst

```diff
@@ -539,6 +539,12 @@ class`. In addition, it provides a few more methods:
 
     .. versionadded:: 3.2
 
+.. method:: int.as_integer_ratio()
+
+  Return a pair of integers whose ratio is exactly equal to the original integer
+  and with a positive denominator. The integer ratio of integers (whole numbers)
+  is always the integer as the numerator and 1 as the denominator.
+
```

Comment:
A `versionadded` directive needs to be added. Also, the [developer's guide](https://devguide.python.org/documenting/#use-of-whitespace) states that reST files should use an indentation of 3 spaces.